ParcelDistributionCenter/ParcelDistributionCenter.Web/Views/Packages/DisplaySinglePackage.cshtml

```diff
@@ -4,6 +4,13 @@
     ViewData["Title"] = "Package added";
 }
 
+@if (TempData["Message"] != null)
+{
+        <div class="alert @TempData["MessageClass"]">
+        @TempData["Message"]
+        </div>
+}
```

Comment:
A może ten fragment dodać do "Shared"? Tak żeby to działało na wszystkich widokach?